Common/Log.h

```diff
@@ -83,7 +83,8 @@ void GenericLog(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type,
 		;
 bool GenericLogEnabled(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type);
 
-#if defined(_DEBUG)
+// Exception for Windows - enable more log levels in release mode than on other platforms.
```

Comment:
I wonder if this might make sense everywhere or at least non-mobile.  Debug logging has a cost, but romhackers may benefit from it and don't always use Windows.  Although, there is WINE and debugging still mostly requires Windows, so definitely not an immediate problem.

-[Unknown]